src/components/misc/Carousel/Carousel.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+
+class Carousel extends Component {
+
+    state = {
+      initialPosition: 0
+    }
+
+    backwards () {
+      if (this.state.initialPosition > 0) {
+        this.setState({
+            initialPosition: this.state.initialPosition - 1 
+        })
+      }
+    }
+
+    forward () {
+      if (this.state.initialPosition < 3){
+        this.setState({
+            initialPosition: this.state.initialPosition + 1
+        })
+      }    
+    } 
+
+
+  render() {
+    return (
+      <div>
+        <img 
+        src={this.props.images[this.state.initialPosition]}
+        alt="Carousel"
+        />
+        <div>
+            <button onClick={() => this.forward()}>Forward</button>
```

Comment:
si no vas a pasarle parámetros a la función entonces mejor así:
```suggestion
            <button onClick={ this.forward }>Forward</button>
```